presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/presto/etc/catalog/cassandra.properties

```diff
@@ -0,0 +1 @@
+connector.name=jmx
```

Comment:
Include this comment:
https://github.com/prestosql/presto/blob/cd65c0552816d36309ec24b8c0817831a49afdfd/presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/presto/etc/catalog/hive.properties#L1-L3